ui/ui-expenses/src/components/file/file.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styles from './file.styles'
+
+const File = ({ name, id }) => (
+  <a href={`https://drive.google.com/uc?export=view&id=${id}`}>
+    <img className={styles.img} src={`https://drive.google.com/uc?export=view&id=${id}`} alt={name} />
```

Comment:
alt attribute not really useful.
Is it a need for you to see the uploaded image ? In first version, the image was hidden.
If this is needed, we have to think about a way to include it in the interface.

Reply:
btw, I think this is not a V1 feature